src/clj/prerenderer/core.clj

```diff
@@ -10,16 +10,19 @@
 (defn ensure-javascript-exists
   ([js-engine] (ensure-javascript-exists js-engine false))
   ([js-engine notify-about-file-appearing]
-   (if (not (.exists (io/as-file (:path js-engine))))
-     (let [message (str "File " (:path js-engine) " for prerendering is not present. Did you compile your JavaScript? 'lein cljsbuild auto' maybe?")]
-       (if (:wait js-engine)
-         (do
-           (println message "Waiting until it appears...")
-           (Thread/sleep 100)
-           (recur js-engine true))
-         (throw (Exception. message))))
-     (when notify-about-file-appearing
-       (println "File" (:path js-engine) "appeared. Pfiuuu!")))))
+   (let [path (:path js-engine)
+         working-directory (:working-directory js-engine ".")
+         full-path (.getAbsolutePath (java.io.File. working-directory path))]
```

Comment:
Are there any chances of getAbsolutePath failing or giving surprising results here?